README.md

```diff
@@ -4,9 +4,22 @@ ramda-fantasy
 [Fantasy Land][1] compatible types for easy integration with [Ramda][2].
 
 ## Project status
-This project is in alpha status. The implementation of the Fantasy Land spec should be *mostly*
-stable. Any methods outside of the Fantasy Land spec are subject to change. The types also have
-not undergone thorough testing/use yet.
+The Ramda team has decided that this library is no longer serving the JavaScript FP community well. It has not kept up with changes to the Fantasy Land spec, and many of the data types lack useful features available in other libraries.
+
+There are, though, a number of people using ramda-fantasy. We wish to wind down this project without affecting these users, so of course the npm releases will remain available indefinitely. 
+
+Here are links to replacement libraries that we can recommend:
+
+    Maybe: sanctuary-js/sanctuary-maybe†
+    Either: sanctuary-js/sanctuary-either†
+    Future: fluture-js/Fluture
+    State: fantasyland/fantasy-states
+    Tuple: fantasyland/fantasy-tuples
+    Reader: fantasyland/fantasy-readers
+    IO: fantasyland/fantasy-io
+    Identity: fantasyland/fantasy-identities
+
+† Currently only available as part of Sanctuary.
```

Comment:
rather than listing particular ADTs, I'd rather see links to the libraries, e.g., 

* Fantasy-Land
* Fluture
* Folktale
* Sanctaury

with appropriate links.